### ISSUE TITLE
Incorrect ARIA attribute in the `BitModal` component

### ISSUE THREAD

**Issue by msynk** [bitfoundation/bitframework#1941]
Currently, there is a typo in one of the ARIA attributes applied to the role DIV inside the BitModal component.
The incorrect attribute is `aria-model` which should be corrected to `aria-modal`.
